infrastructure/prime-app-ephemeral-template.yml

```diff
@@ -531,7 +524,10 @@ objects:
               - createdb -O $(APPUSER) prime-${SVC_NAME}
             env:
               - name: PGHOST
-                value: dev-patroni
+                valueFrom:
+                  configMapKeyRef:
+                    name: database-cm
+                    key: database-host
               - name: PGDATABASE
                 value: postgres
```

Comment:
Rather than `postgres`, should this be `prime-${SVC_NAME}`?

Reply:
This is part of the database creation script. The database has to be created with the superuser.

Reply:
Can you please add a comment re: that?

Reply:
I'm ok with this at the moment. Lots of this will change with helm.